src/runtime/toRuntime.ts

```diff
@@ -0,0 +1,30 @@
+import { Logger, Scope, Layer, Effect, Runtime, Exit } from "effect"
+
+// Define a configuration layer
+const appLayer = Logger.replace(
+  Logger.defaultLogger,
+  Logger.make(({ message }) => console.log(message))
+)
+
+// Create a scope for resources used in the configuration layer
+const scope = Effect.runSync(Scope.make())
+
+// Transform the configuration layer into a runtime
+const runtime = Effect.runSync(
```

Comment:
Sometimes layers require async construction. This will raise an error in that case - something to take note of.